src/reducers/starWarsReducer.js

```diff
@@ -1,13 +1,34 @@
-import /* we need our action types here*/ "../actions";
+import {FETCH_START, FETCH_SUCCESS, FETCH_FAILURE} from "../actions";
 const initialState = {
-  characters: []
+  characters: [],
+  fetching: false,
+  error: ''
   // Array characters, Boolean fetching, null error.
 };
 export const charsReducer = (state = initialState, action) => {
   switch (action.type) {
     // Fill me in with the important reducers
     // action types should be FETCHING, SUCCESS and FAILURE
     // your switch statement should handle all of these cases.
+    case FETCH_START:
+      return {
+        ...state,
+        fetching: true,
+        error: ''
```

Comment:
When there is no error, keep it set to `null` instead of an empty string `""` - `null` states more declaratively that there is no error, as opposed to having an 'error' that is composes of empty text.